mlflow/langchain/__init__.py

```diff
@@ -315,6 +316,8 @@ def load_retriever(persist_directory):
     if metadata is not None:
         mlflow_model.metadata = metadata
 
+    streamable = isinstance(lc_model, lc_runnables_types())
```

Comment:
are there any runnable that can't stream?

Reply:
they all support stream